### ISSUE TITLE
`gsub': can't convert nil into String (TypeError)

### ISSUE THREAD

**Issue by mjhea0** [ruhoh/ruhoh.rb#223]
``` shell
$ rvm use ruby-1.9.3-p194
Using /usr/local/rvm/gems/ruby-1.9.3-p194
$ bundle update
Fetching gem metadata from https://rubygems.org/..........
Fetching gem metadata from https://rubygems.org/..
Resolving dependencies...
Using directory_watcher (1.4.1)
Using hike (1.2.3)
Using mini_portile (0.5.1)
Using multi_json (1.8.0)
Using mustache (0.99.4)
Using nokogiri (1.6.0)
Using psych (1.3.4)
Using rack (1.5.2)
Using redcarpet (2.3.0)
Using ruhoh (2.4)
Using sass (3.2.10)
Using tilt (1.4.1)
Using sprockets (2.10.0)
Using bundler (1.3.5)
Your bundle is updated!
$ ruhoh posts draft
Using theme: "twitter"
=> Oh boy! Asset pipeline enabled by way of sprockets =D
/usr/local/rvm/gems/ruby-1.9.3-p194/gems/ruhoh-2.4/lib/ruhoh/resources/pages/client.rb:68:in `gsub': can't convert nil into String (TypeError)
    from /usr/local/rvm/gems/ruby-1.9.3-p194/gems/ruhoh-2.4/lib/ruhoh/resources/pages/client.rb:68:in `render_scaffold'
    from /usr/local/rvm/gems/ruby-1.9.3-p194/gems/ruhoh-2.4/lib/ruhoh/resources/pages/client.rb:79:in `block in create'
    from /usr/local/rvm/gems/ruby-1.9.3-p194/gems/ruhoh-2.4/lib/ruhoh/resources/pages/client.rb:79:in `open'
    from /usr/local/rvm/gems/ruby-1.9.3-p194/gems/ruhoh-2.4/lib/ruhoh/resources/pages/client.rb:79:in `create'
    from /usr/local/rvm/gems/ruby-1.9.3-p194/gems/ruhoh-2.4/lib/ruhoh/resources/pages/client.rb:35:in `draft'
    from /usr/local/rvm/gems/ruby-1.9.3-p194/gems/ruhoh-2.4/lib/ruhoh/client.rb:49:in `initialize'
    from /usr/local/rvm/gems/ruby-1.9.3-p194/gems/ruhoh-2.4/bin/ruhoh:34:in `new'
    from /usr/local/rvm/gems/ruby-1.9.3-p194/gems/ruhoh-2.4/bin/ruhoh:34:in `<top (required)>'
    from /usr/local/rvm/gems/ruby-1.9.3-p194/bin/ruhoh:23:in `load'
    from /usr/local/rvm/gems/ruby-1.9.3-p194/bin/ruhoh:23:in `<main>'
```

**Comment by wzpan** [comment]
It has been fixed in #221 .

**Comment by plusjade** [comment]
Pushing out 2.5 this weekend guys!
